CppHeaderParser/test/TestSampleClass.h

```diff
@@ -90,9 +90,9 @@ namespace Alpha
 			///
 			typedef enum
 			{
-				RI_ZERO,
-				RI_ONE,
-				RI_TWO
+				RI_ZERO, /// item zero
+				RI_ONE,  /** item one */
+				RI_TWO   //!< item two
```

Comment:
Could we also test variants without Doxygen comments and variants with Doxygen comments before them?

Reply:
Covered elsewhere, but sure.